app/views/profile/_header.html.slim

```diff
@@ -3,7 +3,7 @@
     h2.h3.m0.center.sm-left-align
       .sm-hide.regular.sans-serif.inline
         span
-          = t('shared.nav_auth.welcome')
+          = t('profile.nav_auth.welcome')
```

Comment:
seems like this is in may partials that are not named `nav_auth` -- should we just name the translation `profile.welcome` ?